mathesar_ui/src/stores/tables.ts

```diff
@@ -108,7 +108,9 @@ export async function refetchTablesForSchema(
   }
 }
 
-let preload = true;
+// Since we are not landing on the first schema page by default
+// hence preloading the first schema data won't make any sense
+let preload = false;
```

Comment:
We need to identify this boolean based on whether tables can be preloaded. This information depends on the first url the user visits. Refer https://github.com/centerofci/mathesar/pull/1634#pullrequestreview-1107277826.

**We need to handle this scenario for explorations as well.**